### PR TITLE
New section - Message bus configuration 

### PR DESCRIPTION
**Pull request by sunithasomanna**
Added instructions for configuring message bus (in Plugin Developer's Guide).
Added description for the configuration parameters.

### CODE REVIEW

File: plugin-redfish/README.md [hunk 1]
```diff
@@ -197,6 +199,38 @@ Each plugin implements API services conforming to specific standards targeted at
...
+The common messaging architecture is used to forward events received from the plugin layer to the upper layers. During the run-time, the plugin uses either Kafka or the Redis stream service as the event message bus.
+
+Following are the parameters to be configured in the plugin configuration file.
+
+```
+"MessageBusConf": {
+"MessageQueueConfigFilePath": "",
+"MessageBusAddress": "localhost",
+"MessageBusPort": "6379",
+"HASet": "redisSentinel",
```
Comment (Bharath-KKB) [inline plugin-redfish/README.md]: We should not have parameters specific to a message bus type in this config. MessagebusAddress/MessagePort/HASet should be removed and be part of the messagebus configfilepath. As the developer to fix this first

File: plugin-redfish/README.md [hunk 2]
```diff
@@ -197,6 +199,38 @@ Each plugin implements API services conforming to specific standards targeted at
...
 
+### Message bus configuration
+
+The common messaging architecture is used to forward events received from the plugin layer to the upper layers. During the run-time, the plugin uses either Kafka or the Redis stream service as the event message bus.
+
+Following are the parameters to be configured in the plugin configuration file.
+
+```
+"MessageBusConf": {
+"MessageQueueConfigFilePath": "",
```
Comment (Bharath-KKB) [inline plugin-redfish/README.md]: MessageQueueConfigFilePath should be changed to Message**Bus**ConfigFilePath. Get the team to fix this before the docs are set right

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
Address the two comments.

File: plugin-redfish/README.md (hunk 1 above)
Comment (sunithasomanna) [reply plugin-redfish/README.md]: Fixed in doc.

File: plugin-redfish/README.md (hunk 2 above)
Comment (sunithasomanna) [reply plugin-redfish/README.md]: Informed Jeevan. Team will be fixing it in Jan. Once that is fixed, I'll update the document. Keeping this PR open until then!

Comment (sunithasomanna) [reply plugin-redfish/README.md]: Updated the parameter name `MessageQueueConfigFilePath` with `MessageBusConfigFilePath` after Dev update.

**Review by Bharath-KKB — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**